README.md

```diff
@@ -73,6 +73,17 @@ loaded_model_from_hub = INCModelForSequenceClassification.from_pretrained(
 
 You can load many more quantized models hosted on the hub under the Intel organization [`here`](https://huggingface.co/Intel).
 
+#### One-click acceleration with launcher
+
+Users can use our Python launcher design to run the Python model code as it is with automatic enabling of the optimization. For example,
```

Comment:
```suggestion
Dynamic quantization and ONNX export can be used through the Optimum Intel command-line:
```
(it would be great to have the ONNX export by default after `neural-compressor` next release)